.agents/skills/superdesign/SKILL.md

```
@@ -0,0 +1,83 @@
+3. **Set design system**
+4. **Help me improve design of X**
+
+# Init: Repo Analysis
+
+When `.superdesign/init/` directory doesn't exist or is empty, you MUST automatically:
+
+1. Create the `.superdesign/init/` directory
+2. Fetch the init prompt from the URL below
+3. Follow the prompt instructions to analyze the repo and write context files
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)